eclass/openib.eclass

```diff
@@ -7,11 +7,19 @@
 # @AUTHOR:
 # Author: Alexey Shvetsov <alexxy@gentoo.org>
 # @BLURB: Simplify working with OFED packages
+# @SUPPORTED_EAPIS: 5 6 7
 
-inherit eutils rpm versionator
+case ${EAPI:-0} in
+	[56]) inherit eutils rpm versionator ;;
+	7) inherit rpm ;;
```

Comment:
I think you cannot support EAPI 7 without additional changes.

For example, there is this line in global scope which will need versionator.eclass:
```
OFED_BASE_VER=$(get_version_component_range 1-3 ${OFED_VER})
```
(There may be more, I haven't checked.)

Reply:
You're right.